meson.build

```diff
@@ -176,13 +176,6 @@ if bash_completion_dep.found()
   subdir('toolbox'/'bash_completion.d')
 endif
 
-# man-pages: run the Makefile target 'make gen-man-pages' to update the man-pages
```

Comment:
How about, instead of removing the option of not installing man-pages, then leave it as is and instead replace``man/meson.build`` with:

```meson
if get_option('tests')
  subdir('tests')
endif
if get_option('tools')
  subdir('tools')
endif
if get_option('examples')
  subdir('examples')
endif
```

Reply:
Similar could be done for the completions.

I propose this, since it would be less of a functional change, in case the meson-options were still usable, then it would not be a 'build-api-breaking-change'. But just a ``fix(meson): ...``.

Anyway, either direction is fine with me.